indexing-service/src/main/java/org/apache/druid/indexing/seekablestream/supervisor/SeekableStreamSupervisorSpec.java

```diff
@@ -88,6 +88,9 @@ public SeekableStreamSupervisorSpec(
       @JacksonInject SupervisorStateManagerConfig supervisorStateManagerConfig
   )
   {
+    Preconditions.checkArgument(supervisorStateManagerConfig.isEnableIdleBehaviour()
+                                || !ingestionSchema.getIOConfig().isEnableIdleBehaviour(),
+                                "Idle behaviour is disabled on Overlord. It cannot be enabled per Supervisor.");
```

Comment:
This may break an existing supervisor's ingestion when the cluster-wide setting is turned off

Reply:
Reverted it. Thanks.